cmd/yoke/cmd_takeoff.go

```diff
@@ -130,6 +135,27 @@ func TakeOff(ctx context.Context, params TakeoffParams) error {
 		return ExportToFS(params.Out, params.Release, resources)
 	}
 
+	if params.DiffOnly {
+		revisions, err := kube.GetRevisions(ctx, params.Release)
+		if err != nil {
+			return fmt.Errorf("failed to get revision history: %w", err)
+		}
+		latestAppliedRevision := revisions.History[len(revisions.History)-1]
```

Comment:
Here I believe revisions has a `CurrentResources` method. This is because yoke supports rollbacks to previous versions. So the current revision id is not necessarily the last one created!